pwnlib/libcdb.py

```diff
@@ -126,7 +131,25 @@ def provider_local_system(hex_encoded_id, hash_type):
         return local_libc.data
     return None
 
-PROVIDERS = [provider_local_system, provider_libcdb, provider_libc_rip]
+# Offline search https://github.com/niklasb/libc-database for hash type
+def provider_local_database(hex_encoded_id, hash_type):
+    if not context.local_libcdb:
+        log.error("`context.local_libcdb` could not be null or empty string.")
```

Comment:
I think you'd expect the local libc-database integration to just be turned off when you set the path to a falsely value instead of requiring a path. Can we just return None here?